ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.6.yaml

```diff
@@ -0,0 +1,248 @@
+  assisted-installer-controller:
+    name: ocm-2.6
+    namespace: edge-infrastructure
+    tag: assisted-installer-controller
+  assisted-service:
+    name: ocm-2.6
+    namespace: edge-infrastructure
+    tag: assisted-service
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: "4.10"
+    namespace: ocp
+    tag: hypershift-operator
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |
+    FROM base
+    COPY . .
+  from: src
+  to: assisted-test-infra
+- dockerfile_path: Dockerfile.assisted-test-infra
+  to: assisted-test-infra-internal
+promotion:
+  name: ocm-2.6
+  namespace: edge-infrastructure
+releases:
+  initial:
+    integration:
+      name: "4.9"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.9"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: lint
+  commands: |
+    make lint
+  container:
+    from: assisted-test-infra-internal
+- as: e2e-metal-assisted-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        SERVICE_BASE_REF=release-ocm-2.6
+      TEST_SUITE: full
+    workflow: assisted-baremetal
+- as: e2e-metal-assisted-tpmv2
+  optional: true
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        SERVICE_BASE_REF=release-ocm-2.6
+        DISK_ENCRYPTION_ROLES=all
+        DISK_ENCRYPTION_MODE=tpmv2
+    workflow: assisted-baremetal
+- as: e2e-metal-assisted-tpmv2-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        SERVICE_BASE_REF=release-ocm-2.6
+        DISK_ENCRYPTION_ROLES=all
+        DISK_ENCRYPTION_MODE=tpmv2
+      TEST_SUITE: full
+    workflow: assisted-baremetal
+- as: e2e-metal-assisted-cnv
+  optional: true
+  run_if_changed: .*operator.*
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        SERVICE_BASE_REF=release-ocm-2.6
+        OLM_OPERATORS=cnv
+    workflow: assisted-baremetal
+- as: e2e-metal-assisted-cnv-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        SERVICE_BASE_REF=release-ocm-2.6
+        OLM_OPERATORS=cnv
+      TEST_SUITE: full
+    workflow: assisted-baremetal
+- as: e2e-metal-assisted-ocs
+  optional: true
+  run_if_changed: .*operator.*
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        SERVICE_BASE_REF=release-ocm-2.6
+        OLM_OPERATORS=ocs
+    workflow: assisted-baremetal
+- as: e2e-metal-assisted-ocs-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        SERVICE_BASE_REF=release-ocm-2.6
+        OLM_OPERATORS=ocs
+      TEST_SUITE: full
+    workflow: assisted-baremetal
+- as: e2e-metal-assisted-day2
```

Comment:
We now have a ztp-based day2 job (at the very least with SNO expansion) so it can be replaced with it